tests/rptest/tests/rbac_test.py

```diff
@@ -907,6 +927,8 @@ def test_role_survives_restart(self):
 
         assert r.name == rand_role
 
+        self._wait_for_everything_snapshotted(self.redpanda.nodes, admin)
```

Comment:
I wonder if it's possible for the snapshot to sneak in between the last command `update_role_members` and this line. Maybe this should go above the wait on 922?

Reply:
hmm, yeah I'll play with it a bit locally